examples/import-html/html-fragment-footer.html

```diff
@@ -0,0 +1,5 @@
+
+<footer>
+  This footer can be re-used across pages without any server or build code.
+</footer>
+
```

Comment:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#729795 
https://robots.thoughtbot.com/no-newline-at-end-of-file 

what's the justification for removing it again? if you want to enforce a style rule like that it should get documented somewhere first, esp since there is a conflicting guidance in popular usage.